### PR TITLE
The list of all oeprators is created to avoid any conflicts

### PR DESCRIPTION
**Pull request by mozga-intel**
The pull-request shows the list of the incorporated/unincorporated operators. We'll avoid a proliferation of underspecified conflicts after the merge. 

CC: @baojun-nervana

### CODE REVIEW

**Comment by luotao1** [comment]
why not use https://github.com/PaddlePaddle/Paddle/pull/14805#discussion_r245865774 to avoid conflict?

**Comment by mozga-intel** [comment]
@luotao1 it looks good, I think the code presented by me is clear and simpler than the auto-generation process. However If you prefer to have a PP-like header generator, I will be able to undertake the job of preparing the code according to your suggestions.

**Comment by luotao1** [comment]
We prefer a header generator, since both paddle ops and tensorrt ops are auto-generated.

**Comment by mozga-intel** [comment]
@luotao1 Could you please look at this pull-request.

File: paddle/fluid/operators/ngraph/ops/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+file(GLOB LIST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*.h")
+set(pass_file ${PADDLE_BINARY_DIR}/paddle/fluid/operators/ngraph/ngraph_ops.h)
+file(WRITE ${pass_file} "// Generated by the /paddle/fluid/operators/ngraph/ops/CMakeLists.txt.  DO NOT EDIT!\n\n")
```
Comment (luotao1) [inline paddle/fluid/operators/ngraph/ops/CMakeLists.txt:4]: remove the file`paddle/fluid/operators/ngraph/ngraph_ops.h`, and  `file(APPEND ${pass_file} "\#pragma once\n")`

File: paddle/fluid/operators/ngraph/ngraph_bridge.cc [hunk 2]
```diff
@@ -32,23 +32,60 @@ std::map<std::string,
                                 std::string, std::shared_ptr<ngraph::Node>>>)>>
     NgraphBridge::NG_NODE_MAP = {
         {"accuracy", NG_OPS::BuildAccuracyNode},
```
Comment (luotao1) [inline paddle/fluid/operators/ngraph/ngraph_bridge.cc]: It's not elegant to list all the NG_NODE_MAP here. Could you generate it automatically?

Comment (mozga-intel) [reply paddle/fluid/operators/ngraph/ngraph_bridge.cc]: Sure, I've been working on this issue, yet.

File: paddle/fluid/operators/ngraph/ops/CMakeLists.txt (hunk 1 above)
Comment (mozga-intel) [reply paddle/fluid/operators/ngraph/ops/CMakeLists.txt:4]: Done.

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**